rts/Game/UI/GuiHandler.cpp

```diff
@@ -2775,7 +2775,7 @@ static inline bool BindUnitTexByString(const std::string& str)
 	return true;
 }
 
-
+/*
```

Comment:
Maybe add a comment on why this is commented, e.g. "Icon handling has been completely reworked, see something" or straight up remove this code